WooCommerce/Classes/ViewRelated/Upgrades/WooPlanCardView.swift

```diff
@@ -0,0 +1,146 @@
+import Yosemite
+import SwiftUI
+
+struct WooPlanCardView: View {
+    let upgradePlan: WooWPComPlan
+    @Binding var selectedPlan: WooWPComPlan?
+
+    private var isSelected: Bool {
+        selectedPlan?.id == upgradePlan.id
+    }
+
+    private var isPopular: Bool {
```

Comment:
Do we have a view model for this? It seems like all this logic belongs there, and can be tested as well.

Reply:
Good point, I renamed the property to `shouldDisplayIsPopularBadge` and moved it to the ViewModel here: 2fe488acd833e42ecc880d72f6057e735a865612